packages/geo/package.json

```diff
@@ -1,12 +1,10 @@
 {
   "name": "@accelint/geo",
-  "version": "0.1.3",
+  "version": "1.0.0",
```

Comment:
We may want to change this back to `0.0.0` as the `major` changeset would actually bump this to `2.0.0`. Additionally, for a initial release, we have been going with a `minor` version to sort of denote a "beta" in a way. e.g. `0.1.0`

Reply:
Could certainly do that as well! Given we are starting from "scratch" a little bit with the new implementation, I am also okay with a `0.1.0`

Reply:
Actually, just remembered we were publishing this already to NPM so we will need to keep the current version of `0.1.3`